src/app/mini-games/sprint/sprint-mini-game.component.html

```diff
@@ -1 +1,79 @@
-<p>sprint-mini-game works!</p>
+<div class="background-container">
+  <div *ngIf="game.gameState === 'PLAY'" class="game__counter">
+    <app-counter
+      [count]="59"
+      [isPaused]="game.isPaused"
+      [spinnerColor]="'warn'"
+      (countIsOver)="gameOver()"
+    ></app-counter>
+  </div>
+
+  <div class="game-container">
+    <mat-icon class="game__close" (click)="openCloseDialog()">close</mat-icon>
+
+    <ng-container *ngIf="game.gameState === 'SETUP'">
```

Comment:
@Suggestion
You have 'SETUP' in enum so it's better to use it

Reply:
See the comment above.